packages/server/src/radpack.js

```diff
@@ -225,6 +227,7 @@ export default class extends Radpack {
           nextUrls = [...options.urls];
           this._setRegister(index, registers);
           this._resetExports();
+          options.done(nextUrls, this);
```

Comment:
might be useful to use an options bag here, in case we decide to add other useful data later, like what changed, etc.